### PR TITLE
Add Doughnut.app v1.1.1

### PR DESCRIPTION
**Pull request by dyerc**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/doughnut.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "doughnut" do
+  version "1.1.1,58"
+  sha256 "e50a64f6325bc88378f04848213ff745360d6a84a7533e7ad93b5abe53015ba9"
+
+  url "https://github.com/dyerc/Doughnut/releases/download/v#{version.before_comma}/Doughnut-#{version.before_comma}.dmg"
```
Comment (neutric) [inline Casks/doughnut.rb:5]: Please change `#{version.before_comma}` to the new and shiny `#{version.csv.first}`.

File: Casks/doughnut.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  name "Doughnut"
+  desc "Podcast client (podcatcher)"
+  homepage "https://github.com/dyerc/Doughnut"
+
+  livecheck do
+    url "https://raw.githubusercontent.com/dyerc/Doughnut/master/appcast.xml"
+    strategy :sparkle
+  end
+
+  # depends_on macos: ">= :catalina"
```
Comment (neutric) [inline Casks/doughnut.rb:15]: Uncomment or delete, please.

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @dyerc!

File: Casks/doughnut.rb (hunk 2 above)
Comment (neutric) [reply Casks/doughnut.rb]: Since you chose to delete: what is the oldest OS version supported?

**Comment by bevanjkay** [comment]
Thank you @dyerc!

**Pull request merged**